src/lib.rs

```diff
@@ -118,31 +121,57 @@ pub async fn subscribe_stats(
                     weight: total_weight,
                     num_extrinsics: stats.num_extrinsics,
                     max_pov: POV_MAX,
-                    max_weight: max_block_weights.max_block,
+                    max_weight: max_block_weights.max_block.ref_time,
```

Comment:
I think we should make `max_weight` and `weight` contain the full `Weight` type, not just the `ref_time`. This is useful for what we are measuring.

Reply:
I have added full Weight type